src/components/Viewer.jsx

```diff
@@ -66,11 +76,26 @@ export class Viewer extends Component {
     this.setState({ isImageLoading: false })
   }
 
+  toggleToolbar () {
+    // invert the current state
+    const hidden = !this.state.hideToolBar
+    this.setState({ hideToolBar: hidden })
+    // if the toolbar is now visible, reset the hide delay
+    if (!hidden) this.hideToolbarAfterDelay()
+  }
+
+  hideToolbarAfterDelay () {
+    clearTimeout(this.hideToolbarTimeout)
+    this.hideToolbarTimeout = setTimeout(() => {
+      this.setState({ hideToolBar: true })
+    }, TOOLBAR_HIDE_DELAY)
+  }
```

Comment:
sometimes you write `ToolBar` sometimes `Toolbar`. Could we be more consistent? 😉